setup.py

```diff
@@ -97,7 +97,7 @@ def get_current_wheel_details():
         },
         "Linux": {
             "x86_64": {
-                "wheel": "manylinux1_x86_64",
+                "wheel": "linux_x86_64",
```

Comment:
I'm working to get manylinux working, but for now we are much _closer_ to the `linux_x86_64` requirements.  Neither are technically right yet, but it's close.